lib/AST/ASTImporter.cpp

```diff
@@ -6423,9 +6423,14 @@ Expr *ASTNodeImporter::VisitMemberExpr(MemberExpr *E) {
   if (!ToMember && E->getMemberDecl())
     return nullptr;
 
-  DeclAccessPair ToFoundDecl = DeclAccessPair::make(
-    dyn_cast<NamedDecl>(Importer.Import(E->getFoundDecl().getDecl())),
-    E->getFoundDecl().getAccess());
+  auto ToDecl =
```

Comment:
`auto *`? For some reason in LLVM we tend to write the asterisk. I am not sure why though.